.do/db-cluster/databases.tf

```diff
@@ -0,0 +1,34 @@
+resource "digitalocean_database_cluster" "postgres" {
+  for_each = local.environments
```

Comment:
We don't need `for` here. The same template but you just recieve the env name by env variable.

What you call  `each.key` in the entire document can be called `var.environment`